cli/azd/pkg/auth/manager.go

```diff
@@ -432,14 +437,20 @@ func (m *Manager) LoginWithDeviceCode(
 		return nil, err
 	}
 
-	// Display the message to the end user as to what to do next, then block waiting for them to complete
-	// the flow.
-	fmt.Fprintln(deviceCodeWriter, code.Message())
+	console.Message(ctx, fmt.Sprintf("Start by copying the next code: %s", output.WithBold(code.UserCode())))
+	console.Message(ctx, "Then press enter and continue to log in from your browser...")
+	_ = bufio.NewScanner(console.Handles().Stdin).Scan()
 
+	url := "https://login.microsoftonline.com/common/oauth2/deviceauth"
+	if err := m.launchBrowserFn(url); err != nil {
+		console.Message(ctx, fmt.Sprintf("Error launching browser. Manually go to: %s", url))
+	}
+	console.Message(ctx, "Now waiting for user to complete authentication...")
```

Comment:
```suggestion
	console.Message(ctx, "Waiting for you to complete authentication in the browser...")
```

?

Maybe just dropping the "Now" is fine.  It feels a little weird with that prefix to me for some reason, maybe @Austinauth has an idea here?  I think "the user" is a little weird here as well, 'the user' is the one reading this message.  Maybe we just use "you"?

Reply:
I was counting on having some future UX revisions for this in the future.  I was thinking about displaying the spinner while we wait for the user to complete login...  or just blocking with no other messages...  I don't have a string opinion. 
I will apply these suggestions and we can wait on @Austinauth recommendations, which can be done in the future